src/renderer/env-and-consts.ts

```diff
@@ -10,8 +10,8 @@ export const PAGE_STATES_IN_ORDER: PageState[] = [
   'welcome',
   'open-project',
   'configure-recording-set',
-  'settings',
   'recording',
+  'settings',
```

Comment:
Since the settings page is now out of the normal flow of going forward and backward, maybe remove it from this array?

Reply:
It might be consistent and convenient to keep using index to switch?

Reply:
Let's deal with this in another PR then.